assets/js/modules/reader-revenue-manager/components/common/PublicationOnboardingStateNotice.test.js

```diff
@@ -120,6 +135,17 @@ describe( 'PublicationOnboardingStateNotice', () => {
 			expect( container.firstChild ).toHaveClass(
 				'googlesitekit-publication-onboarding-state-notice'
 			);
+
+			// eslint-disable-next-line require-await
+			await act( async () => {
+				fireEvent.click( getByText( ctaText ) );
+			} );
```

Comment:
Hmm, I didn't notice this the first time round, but it doesn't look like this needs to be async either.

More importantly though - as per my previous [comment](https://github.com/google/site-kit-wp/pull/9179#pullrequestreview-2240750196), the issue still needs a QAB. Please can you add one?

```suggestion
			act( () => {
				fireEvent.click( getByText( ctaText ) );
			} );
```